labelbox/schema/asset_attachment.py

```diff
@@ -19,6 +19,8 @@ class AttachmentType(Enum):
         TEXT = "TEXT"
```

Comment:
Should we add a deprecation warning or at least a comment here?

Reply:
I'll create a new ticket to remove this attachment type and in the description I'll specify the exact place where it needs to be removed.

When it comes to warning, maybe we can do it in a follow-up ticket?

Reply:
added a warning https://github.com/Labelbox/labelbox-python/pull/660/commits/db2a6227a5600420637c98f603c7706f28781988